src/components/JobsTrainingContent/index.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import { Container, Row, Col } from "react-bootstrap";
+import "./index.scss";
+import { useTranslation } from "react-i18next";
+import OpportunitiesCard from "../OpportunitiesCard";
+import SearchBar from "../SearchBar";
+
+const JobsTrainingContent = () => {
+  const { t } = useTranslation();
+  const [translationKey, setTranslationKey] = useState("jobOpportunities");
+  const cardsArr = [...t(`home.${translationKey}`, { returnObjects: true })];
```

Comment:
technically the variable on line 12 is also a cardsArr. so you want a more precise variable here to express what this is. otherwise when i see something called "cardsArr" followed by something called "cards" i ask myself, is the second one not also an arr?

Reply:
I actually used cards and ardsArr to match the JobsTrainingHome, I will change the naming.